DigitalInnovationHub/examples/example-normalized.json

```diff
@@ -10,16 +10,23 @@
 		"value": "https://www.sample-dih.com/"
 	},
 	"logo": {
-		"type": "StructuredValue",
-		"value": {
-			"base64Data": "iVBORw0KGgM3ceusb/AAAAAElFTkSuQmCC",
-			"fileName": "logo.png"
-		}
+		"type": "URL",
+		"value": "https://www.sample-dih.com/logo.png"
 	},
 	"description": {
 		"type": "Text",
 		"value": "Digital Innovation Hubs are one-stop-shops that help companies to become more competitive with regard to their business/production processes"
 	},
+	"location": {
+		"type": "GeoProperty",
```

Comment:
this is ngsi-ld in a ngsiv2 payload (eventually it would work) but it does not look like a ngsiv2 payload